data/settings_mapping.json

```diff
@@ -162,12 +162,12 @@
           "row_span": [6,6,9],
           "settings": [
             "starting_age",
+            "mix_entrance_pools",
+            "decouple_entrances",
```

Comment:
While I agree that having a random dropdown in the middle of checkboxes isn't ideal, I don't think we should have mixed pools and decouple above the ER options, since someone reading settings from top to bottom would probably be pretty confused at first.

Reply:
I moved the options down to the bottom above the MQ dungeon options and below the spawn ER option. I also swapped the order for decoupled/mixed to keep the ER toggles together. I'm not sure if that's desirable as decoupled is much more niche right now than mixed pools, assuming least-used options would go at the bottom.

Reply:
Understood. I reverted my changes to the order.